category/tests.py

```diff
@@ -1,3 +1,31 @@
-# from django.test import TestCase
+from .models import Category
+from django.db.utils import IntegrityError
+import pytest
 
-# Create your tests here.
+CATEGORY_NAME = "category1"
+
+
+@pytest.fixture
+def category1():
+    return Category.objects.create(name=CATEGORY_NAME)
+
+
+@pytest.mark.django_db
+class TestCategoryModel:
+
+    def test_update_category(self, category1):
+        new_category_name = "category2"
+        category1.update(new_category_name)
+        category1.save()
+        updated_category = Category.objects.get(id=category1.id)
+        assert updated_category.name == new_category_name
+
+    def test_save(self):
+        category = Category(name=CATEGORY_NAME)
+        category.save()
+        assert category in Category.objects.all()
+
+    def test_category_with_same_name(self):
+        Category.objects.create(name=CATEGORY_NAME)
```

Comment:
you can use the fixture category1 like:
```suggestion
    def test_category_with_same_name(self, category1):
```